### PR TITLE
skipper as deployment 2nd try

### PR DESCRIPTION
**Pull request by szuecs**
change label selectors to ensure controller-manager/scheduler does not go crazy

Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: cluster/manifests/skipper/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    version: v0.10.122
+    component: ingress
+spec:
+  selector:
+    matchLabels:
+      application: skipper-ingress
+      zone: {{ $region }}{{ $suffix }}
+  template:
+    metadata:
+      name: skipper-ingress-{{ $suffix }}
```
Comment (linki) [inline cluster/manifests/skipper/deployment.yaml]: Does that do anything? We usually don't have this field here.

Comment (szuecs) [reply cluster/manifests/skipper/deployment.yaml]: yes with a,b,c it won't overwrite the deplyoment

Comment (szuecs) [reply cluster/manifests/skipper/deployment.yaml]: changed

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**